Utilities/GenerateDocs/README

```diff
@@ -15,9 +15,17 @@ the SwigDocUpdate.sh script can be run to convert the SimpleITK
 Doxygen XML into Swig .i files so that when SimpleITK is built, inline
 documenation for some wrapped languages will be available.
 
+Building the documentation target from the SuperBuild framework requires:
+        The correct cmake flags: -DBUILD_DOXYGEN:BOOL=ON -DBUILD_DOCUMENTS:BOOL=ON
+        Run "make Documentation" in the SimpleITK-build directory.
+        This will create the SimpleITK-build/Documentation/xml content.
+
 Important end use scripts (see script files for more details:
         JSONDocUpdate.sh - Copies ITK Doxygen into SimpleITK JSON
-        SwigDocUpdate.sh - Copies SimpleITK Doxygen into Swig .i files
+        SwigDocUpdate.sh - Copies SimpleITK Doxygen into Swig .i files and
+                           R .Rd files.
+
+        SwigDocUpdate requires python, with the regex library.
```

Comment:
Why is the regex library needed over the standard "[re](https://docs.python.org/2.7/library/re.html)" Python module.

Reply:
Evil recursive regular expression for matching bracket pairs.